mycore-viewer/src/test/resources/mycore-home/mycore-viewer/mycore.properties

```diff
@@ -1 +1,10 @@
 MCR.Persistence.Database.Enable=false
+MCR.Resource.Resolver=mets
```

Comment:
Copy-Paste from `mycore-mets/src/test/resources/mycore-home/mycore-mets/mycore.properties`? Name should probably be `viewer` here, or, better, `test`in both cases, to make future copy-pasting simpler.

Reply:
applied suggested changes